Formula/libdatrie.rb

```diff
@@ -0,0 +1,25 @@
+class Libdatrie < Formula
+  desc "Double-Array Trie Library"
+  homepage "https://linux.thai.net/projects/datrie"
+  url "https://github.com/tlwg/libdatrie/releases/download/v0.2.12/libdatrie-0.2.12.tar.xz"
+  sha256 "452dcc4d3a96c01f80f7c291b42be11863cd1554ff78b93e110becce6e00b149"
+
+  depends_on "autoconf" => :build
```

Comment:
It seems possible we might be able to remove all of these dependencies apart from  `doxygen` based on the release tarball, which we're using, having a pre-generated `configure` script already.